host-interaction/mutex/lock-semaphore.yml

```diff
@@ -0,0 +1,14 @@
+rule:
+  meta:
+    name: lock semaphore
+    namespace: host-interaction/mutex
+    author: alyssa.rahman@fireeye.com
+    scope: file
+    examples:
+      - 3db3e55b16a7b1b1afb970d5e77c5d98
+  features:
+    - and:
+      - os: linux
+      - or:
+        - import: sem_timedwait
+        - import: sem_wait
```

Comment:
the linter will complain about a missing empty line at the end of the file, ensure to run `lint.py`/`lint.py --thorough` on all rule files